pages/HC/operations.vue

```diff
@@ -101,44 +126,63 @@ import axios from 'axios'
           los:'',
           sublos:'',
           bottomlos:'',
-          status:'',
+          category:'',
+          status:'false',
           titles:'',
-          region: '',
+          location:'',
           date1: '',
-          date2: '',
-          delivery: false,
-          type: [],
-          resource: '',
-          desc: ''
+          date2: 'false'
+        },
+         rules: {
+          name: [
+            { required: true, message: 'Please input Employee name and surname', trigger: 'blur' },
+           // { min: 3, max: 5, message: 'Length should be 3 to 5', trigger: 'blur' }
+          ],
+          date1: [
+            { type: 'date', required: true, message: 'Please pick a date', trigger: 'change' }
+          ],
+          date2: [
+            { type: 'date', required: true, message: 'Please pick a time', trigger: 'change' }
+          ],
+          los: [
+            { required: true, message: 'Please select activity resource', trigger: 'change'}
+          ],
+          bottomlos: [
+            { required: true, message: 'Please select activity resource', trigger: 'change' }
+          ],
+           titles: [
+            { required: true, message: 'Please select activity resource', trigger: 'change' }
+          ],
+           location: [
+            { required: true, message: 'Please select location', trigger: 'change' }
+          ],
+          
+          category : [
+            { required: true, message: 'Please select category', trigger: 'change' }
+          ]
+
         }
       };
     },
     methods: {
-    
-      CreateMessage() {
-      axios
-        .post("https://pwcdemo-1c4d3.firebaseio.com/users.json", {
-          username_surname:this.sizeForm.name,
-          user_los:this.sizeForm.los,
-          user_sublos:this.sizeForm.sublos,
-          user_bottomlos:this.sizeForm.bottomlos,
-          user_title:this.sizeForm.titles,
-          user_status:this.sizeForm.status,
-          user_hiredate:this.sizeForm.date1
       
-        })
-        .then(this.$message({
-          message: 'Congrats, this user is added.',
-          type: 'success'
+    submitForm(formName) {
+        this.$refs[formName].validate((valid) => {
+        if (valid) {
+         debugger
```

Comment:
The `debugger` statement should not be pushed.

Reply:
The debugger statement deleted. Thank you for your suggestion.